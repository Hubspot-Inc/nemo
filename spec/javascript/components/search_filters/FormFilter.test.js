import React from "react";
import {shallow} from "enzyme";

import Component from "../../../../app/javascript/components/search_filters/FormFilter";

const defaultProps = {
  allForms: [
    {
      id: "1",
      displayName: "One"
    },
    {
      id: "1",
      displayName: "One"
    }
  ],
  selectedFormIds: [
    "2"
  ]
};

it("renders as expected", () => {
  const wrapper = shallow(<Component {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
