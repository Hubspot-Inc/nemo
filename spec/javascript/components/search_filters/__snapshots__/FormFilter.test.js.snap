// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<Fragment>
  <div>
    [{"id":"1","displayName":"One"},{"id":"1","displayName":"One"}]
  </div>
  <div>
    ["2"]
  </div>
</Fragment>
`;
